native/core/src/execution/shuffle/shuffle_writer.rs

```diff
@@ -793,10 +747,8 @@ impl PartitionBuffer {
             let mut repart_timer = metrics.repart_time.timer();
             self.active = new_array_builders(&self.schema, self.batch_size);
             repart_timer.stop();
-
-            mem_diff += self.active_slots_mem_size as isize;
         }
-        Ok(mem_diff)
```

Comment:
no need to return memory size here because we already reserved the memory in this method